packages/components/src/components/breadcrumb/breadcrumb.tsx

```diff
@@ -1,24 +1,29 @@
-import { Component, h, Prop } from '@stencil/core';
+import { Component, h, Element } from '@stencil/core';
 
 @Component({
   tag: 'ifx-breadcrumb',
   styleUrl: 'breadcrumb.scss',
   shadow: true,
 })
 export class Breadcrumb {
-  @Prop() breadcrumbs: { text: string, url: string }[];
+  @Element() el;
+
+  componentDidLoad() { 
+    const breadcrumbItems = this.el.querySelectorAll('ifx-breadcrumb-item')
```

Comment:
Accessing the Shadow DOM of a child component: In the componentDidLoad method, you're directly accessing the shadow DOM of another component (ifx-breadcrumb-item-label). This could be problematic as it breaks the encapsulation principles of Web Components. It's generally a bad practice to access another component's shadow DOM as it should be considered private to that component. It would be better if the ifx-breadcrumb-item-label component exposes an API for this operation. You can use events or properties to communicate with child components.